edifice/install_osx.md

```diff
@@ -31,3 +31,16 @@ have already installed the library from binaries, run the following command:
 ```bash
 brew uninstall ignition-edifice
 ```
+
+## Ignition libraries are not found
```

Comment:
How about adding a Troubleshooting section like in [other versions](https://github.com/ignitionrobotics/docs/blob/master/citadel/install_osx.md#L31-L33)?

Reply:
When you execute `ign` from a source installation you should source the `setup.bash` file which configure the environment variables such us `IGN_CONFIG_PATH`. In this case this tip is just for the binary installation, including this part here I think make more simpler the initial Ignition setup.

Reply:
Got it, makes sense. Thanks!